harness/test/integration/circuits/capacityDataQuery/oneOfEach.circuit.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  CircuitValue,
+  getAccount,
+  getHeader,
+  getReceipt,
+  getSolidityMapping,
+  getStorage,
+  getTx,
+} from "@axiom-crypto/client";
+
+export interface CircuitInputs {
+  blockNumber: CircuitValue;
+  address: CircuitValue;
+}
+
+export const defaultInputs = {
+  "blockNumber": 5000000, //$ account.eoa[0].blockNumber
+  address: "0x83c8c0b395850ba55c830451cfaca4f2a667a983" //$ account.eoa[0].address
+};
+
+export const circuit = async (inputs: CircuitInputs) => {
+  const header = await getHeader(inputs.blockNumber).receiptsRoot();
+  const account = await getAccount(inputs.blockNumber, inputs.address).balance();
+  const storage = await getStorage(inputs.blockNumber, inputs.address).slot(0);
+  const tx = await getTx(inputs.blockNumber, 0).to();
```

Comment:
are you ensuring that the 0th tx in these cases will be in the `all_small` category?